src/main/java/org/verdictdb/jdbc41/VerdictStreamResultSetMetaData.java

```diff
@@ -0,0 +1,243 @@
+package org.verdictdb.jdbc41;
+
+import org.verdictdb.VerdictSingleResult;
+import org.verdictdb.connection.DataTypeConverter;
+
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+
+import static java.sql.Types.INTEGER;
+
+/**
+ * Created by: Shucheng Zhong on 9/14/18
+ * ResultSetMetaData type for stream select query when VerdictStatement.sql() is called.
```

Comment:
Can you tell me why we need this class separately? If there any problem if we simply use the ResultSetMetaData for the first VerdictSingleResult?

Reply:
Basically they are the same. The reason why I create this class is I find it is hard to adding the new column 'seq' to the original ResultSetMetaData.

Reply:
I see. How about adding a column using the VerdictSingleResult (after making a copy)? If possible, I just want to minimize the code redundancy.

Reply:
It may be difficult to do that because VerdictSingleResult use DbmsQueryResult to get info and DbmsQueryResult, JdbcQueryResult for example, is created directly from ResultSet. Since the interface of those class do not support add column, I think it may be hard to do it without changing the interface.